superset/migrations/versions/2023-05-11_12-41_4ea966691069_cross_filter_global_scoping.py

```diff
@@ -0,0 +1,109 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""cross-filter-global-scoping
+
+Revision ID: 4ea966691069
+Revises: 9c2a5681ddfd
+Create Date: 2023-05-11 12:41:38.095717
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "4ea966691069"
+down_revision = "9c2a5681ddfd"
+
+import copy
+import json
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.ext.declarative import declarative_base
+
+from superset import db
+
+Base = declarative_base()
+
+
+class Dashboard(Base):
+    __tablename__ = "dashboards"
+
+    id = sa.Column(sa.Integer, primary_key=True)
+    json_metadata = sa.Column(sa.Text)
+
+
+def upgrade():
+    bind = op.get_bind()
+    session = db.Session(bind=bind)
+    for dashboard in session.query(Dashboard).all():
```

Comment:
Since this is a pretty huge migration, I think we should paginate this. Check migration `9c2a5681ddfd` for an example of a paginated update.